lib/models/billing.coffee

```diff
@@ -0,0 +1,222 @@
+	# resin.models.billing.getPlan().then(function(billingPlan) {
+	# 	console.log(billingPlan);
+	# });
+	#
+	# @example
+	# resin.models.billing.getPlan(function(error, billingPlan) {
+	# 	if (error) throw error;
+	# 	console.log(billingPlan);
+	# });
+	###
+	exports.getPlan = (callback) ->
+		callback = findCallback(arguments)
+
+		request.send
+			method: 'GET'
+			url: '/user/billing/plan'
+			baseUrl: apiUrl
+		.get('body')
+		.asCallback(callback)
+
+	###*
+	# @summary Get the current billing information
+	# @name getBillingInfo
+	# @public
+	# @function
+	# @memberof resin.models.billing
+	#
+	# @fulfil {Object} - billing information
+	# @returns {Promise}
+	#
+	# @example
+	# resin.models.billing.getBillingInfo().then(function(billingInfo) {
+	# 	console.log(billingInfo);
+	# });
+	#
+	# @example
+	# resin.models.billing.getBillingInfo(function(error, billingInfo) {
+	# 	if (error) throw error;
+	# 	console.log(billingInfo);
+	# });
+	###
+	exports.getBillingInfo = (callback) ->
+		callback = findCallback(arguments)
+
+		request.send
+			method: 'GET'
+			url: '/user/billing/info'
+			baseUrl: apiUrl
+		.get('body')
+		.asCallback(callback)
+
+	###*
+	# @summary Update the current billing information
+	# @name updateBillingInfo
+	# @public
+	# @function
+	# @memberof resin.models.billing
+	#
+	# @param {Object} - an object containing a billing info token_id
+	# @fulfil {Object} - billing information
+	# @returns {Promise}
+	#
+	# @example
+	# resin.models.billing.updateBillingInfo({ token_id: 'xxxxxxx' }).then(function(billingInfo) {
+	# 	console.log(billingInfo);
+	# });
+	#
+	# @example
+	# resin.models.billing.updateBillingInfo({ token_id: 'xxxxxxx' }, function(error, billingInfo) {
+	# 	if (error) throw error;
+	# 	console.log(billingInfo);
+	# });
+	###
+	exports.updateBillingInfo = (billingInfo, callback) ->
+		callback = findCallback(arguments)
+
+		request.send
+			method: 'POST'
+			url: '/user/billing/info'
+			baseUrl: apiUrl
+			body: billingInfo
+		.get('body')
+		.asCallback(callback)
+
+	###*
+	# @summary Get the available invoices
+	# @name getInvoices
+	# @public
+	# @function
+	# @memberof resin.models.billing
+	#
+	# @fulfil {Object} - invoices
+	# @returns {Promise}
+	#
+	# @example
+	# resin.models.billing.getInvoices().then(function(billingInfo) {
+	# 	console.log(billingInfo);
+	# });
+	#
+	# @example
+	# resin.models.billing.getInvoices(, function(error, billingInfo) {
+	# 	if (error) throw error;
+	# 	console.log(billingInfo);
+	# });
+	###
+	exports.getInvoices = (callback) ->
+		callback = findCallback(arguments)
+
+		request.send
+			method: 'GET'
+			url: '/user/billing/invoices'
+			baseUrl: apiUrl
+		.get('body')
+		.asCallback(callback)
+
+	###*
+	# @summary Download a specific invoice
+	# @name downloadInvoice
+	# @public
+	# @function
+	# @memberof resin.models.billing
+	#
+	# @param {String} - an invoice number
+	# @fulfil {String|ReadableStream} - string blob on the browser, download stream on node
```

Comment:
This should be `Blob|ReadableStream`, and it's not a 'string blob', I think it's just a 'blob' (if it does have a mime type, I expect it's application/pdf - there's no string data here)